lib/bolt/application.rb

```diff
@@ -413,6 +413,168 @@ def list_plugins
       { plugins: plugins.list_plugins, modulepath: pal.user_modulepath }
     end
 
+    # Applies one or more policies to the specified targets.
+    #
+    # @param policies [String] A comma-separated list of policies to apply.
+    # @param targets [Array[String]] The list of targets to apply the policies to.
+    # @param noop [Boolean] Whether to apply the policies in no-operation mode.
+    # @return [Bolt::ResultSet]
+    #
+    def apply_policies(policies, targets, noop: false)
+      policies = policies.split(',')
+
+      # Validate that the policies are available to the project.
+      unavailable_policies = policies.reject do |policy|
+        @config.policies.any? do |known_policy|
```

Comment:
I don't *think* people will do this, but if you run `be bolt policy apply project::file, -t docker` this stacktraces

Reply:
Good catch. Updated.